.github/workflows/smoke-test.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Check ${{ matrix.target.template }}
         run: |
-          make -C install/kubernetes ${{ matrix.target.name }}
+          make -C install/kubernetes USE_DIGESTS=true ${{ matrix.target.name }}
```

Comment:
I don't think this will work, we'll need to fix up the `FIXME` in the actual Makefile to set `useDigests=true` when the `USE_DIGESTS` variable is set to true.

That'll be the default so we shouldn't need this change here.

Reply:
Oh I see, apologies. I misunderstood the point of `USE_DIGESTS`.

Reply:
Basically this: https://github.com/cilium/cilium/pull/15308

Reply:
Alright, pushed now. Should be good now :crossed_fingers: